### ISSUE TITLE
Estimating sample size for normal outcome

### ISSUE THREAD

**Issue by alexandrosgryparis** [tobiasmuetze/ThreeArmedTrials#1]
Hi, I am trying to obtain a sample size for a 3-arms trial with normally distributed outcome. I am using something like:

power_RET(experiment = c(180,20), reference = c(180,20), placebo =c (160,20), 
          Delta = 0.8, sig_level = 0.025, power = 0.8, 
          allocation = c(1, 1, 1) / 3, 
           distribution = "normal")

I am getting the following error message:
"Error in calc_power_ret.normal(x = x, Delta = Delta, allocation = w, n = n,  : 
  Two parameters must be defined for power related calculations for normal endpoints."

Am I doing something wrong here? How can I make this work?
Thanks

**Comment by tobiasmuetze** [comment]
Thanks. Did you try using the github version of the package (version 1.0-2)? The error is fixed in version 1.0-2. Also, your parameter combination is not located in the parameter space of the alternative hypothesis. The hypothesis is described in detail in the vignette. To make it work with the same effect, try using -180 and -160 as the means.

**Comment by alexandrosgryparis** [comment]
Dear Tobias,
thank you for your reply. I have indeed tried to install the  github version of the package (version 1.0-2) before I emailed you. I couldn't install it, because one of the packages for devtools cannot get installed properly.  Specifically, I am talking about the package R6, which got updated pretty recently (on 14/2/2019) and may have a bug.
Thanks again.Alexandros



    On Saturday, February 16, 2019, 11:21:22 PM GMT+2, tobiasmuetze <notifications@github.com> wrote:  
 
 
Closed #1.
